markdown/articles/2021/08/22/Non-blue-fair-decks.md

```diff
@@ -0,0 +1,429 @@
+You have to learn assess your opponents posibilities, that's not something you'll learn reading this guide. You need to play matches over matches to refine this skill. But i can give you list of usefull tools you propably have in your deck.  
+Fetches are great to play around mana denial. You can "hide" you mana producing lands behind them while thinning your deck.  
+Free Countermagic is your best tool against every form hate spells. From {{Endurance}} over {{Choke}} to {{Sphere of Resistance}}.  
+Artifact mana is super good when you want to end the game fast and you know they don't have ways to mess with it.  
+{{Cavern of Souls}} is an easy counter to Pyroblast.  
+Bounce Spells are the way to go to beat a fast Marit Lage.  
+
+## Mono-Green Cloudpost
+
+### Gameplan
+
+Just slam, and beat Endurance post Doomsday
+
+### Cards to keep in mind
+
+#### Maindeck
+
+<row variant="hand"> {{!Endurance}} {{!Dark Depths}}</row>
+
+#### Sideboard
+
+<row variant="hand"> {{!Mindbreak Trap}}</row>
+
+### Gameplay
+
+#### Pre Doomsday
+
+This will be a short one. Just cast Doomsday asap and play around possible {{Mindbreak Traps}} in sideboarded games. Like in any Dark Depths deck, their plays to creat a 20/20 are super telegraphed.
+
+#### Post Doomsday 
+
+They only have one card to interact with you, {{Endurance}}. You should always have enough time for a pile that beats Endurance and even plays around Mindbreak Trap.
+
+## GW Depths
+
+### Gameplan
+
+Cast Doomsday as soons as possible, and play around Endurance.
+
+### Cards to keep in mind
+
+#### Maindeck
+
+<row variant="hand"> {{!Endurance}} {{!Crop Rotation}} {{!Dark Depths}} {{!Wasteland}} {{!Elvish Reclaimer}}</row>
+
+#### Sideboard
+
+<row variant="hand"> {{!Deafening Silence}} {{!Choke}} {{!Collector Ouphe}}</row>
+
+### Gameplay
+
+#### Pre Doomsday
+
+The matchup plays pretty similar to the Maverick matchup, but you have to focus more on speed. GW Depts can set up a pretty early Dark Depths Combo with it's fast mana and strong tutors, but they are not really a fast combo deck. Most hands can't kill you realiably through countermagic in the early turns of the game.  
+They have no counterplay against you in Game 1 besides pressure and a bit of mana dial with Wasteland, so just slam early. In Game 2 and 3 they propably bring in more mana denial like Collector Ouphe or Choke and some basic combo hate with Deafening Silence. Still nothing a fast Doomsday can't beat.
+
+#### Post Doomsday 
+
+For your pile, evaluate their pressure and build accordingly, but try to play around Endurance if possible.
+
+## The epic storm
+
+### Gameplan
+
+Keep countermagic up and do not pass the turn after Doomsday without protection.
+
+### Cards to keep in mind
+
+#### Maindeck
+
+<row variant="hand"> {{!Orim's Chant}} {{!Veil of Summer}} {{!Echo of Eons}}</row>
+
+#### Sideboard
+
+<row variant="hand"> {{!Grapeshot}}</row>
```

Comment:
Unsure Grapeshot is the issue.....
Relay might be worth a mention (pre DD mostly).

Reply:
I'm unsure, but in my matches against TES Relay was irrelevant.
Grapeshot is a tool to kill you post Doomsday with less mana than Tendrils